aspnetcore/migration/21-to-22.md

```diff
@@ -115,6 +115,10 @@ The following table shows the Docker image tag changes:
 
 Change the `FROM` lines in your *Dockerfile* to use the new image tags in the preceding table's 2.2 column.
 
+## Visual Studio Auto build on browser request with IIS in-process hosting
```

Comment:
This heading should start with an action verb, as the other headings do.

Reply:
Any I-dears? This section is stating a fact of capabilities and not asking the reader to take any specific action. It's just saying "don't expect this to work." What's the action verb/verbal phrase that makes sense in this scenario? 🤔 ...

1. Avoid Visual Studio Auto build on browser request with IIS in-process hosting
1. Build manually in Visual Studio when using IIS in-process hosting
1. Observe regression with Visual Studio Auto build on browser request and IIS in-process hosting
1. Note regression with Visual Studio Auto build on browser request and IIS in-process hosting

🤔 ...... so far, I like number 2 ... shorter. Which one do you like, or should I keep brainstorming?